package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@alpacahq/alpaca-trade-api",
+  "name": "@neeschit/alpaca-trade-api",
```

Comment:
@shlomikushchi  - I have my own package as well, so this why I closed the PR. It was truly an error, but I'm happy to contribute any of these changes if y'all are interested